docs/commands/histogram.md

```diff
@@ -0,0 +1,172 @@
+# histogram
+
+Creates a new table with a histogram based on the column name passed in.
+
+Syntax: `histogram <column_name>  ...args`
+
+### Parameters
+
+* `<column-name>`: name of the column to graph by
+* `args`: column name to give the histogram's frequency column
+
+## Examples
+
+Let's say we have this file `random_numers.csv` which contains 50 random numbers.
+
+**Note**: The input doesn't have to be numbers it works on strings too. Try it out.
+
+```shell
+> open random_numbers.csv
+━━━━┯━━━━━━━━━━━━━━━
+ #  │ random number 
+────┼───────────────
+  0 │ 87 
+  1 │ 46 
+  2 │ 39 
+  
+   ...
+
+ 47 │ 94 
+ 48 │ 61 
+ 49 │ 67 
+━━━━┷━━━━━━━━━━━━━━━
+```
+
+If we now want to see how often the different numbers were generated, we can use the `histogram` function:
+
+```shell
+> open random_numbers.csv | histogram "random number"
+━━━━┯━━━━━━━━━━━━━━━┯━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+ #  │ random number │ frecuency 
+────┼───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────
+  0 │ 10            │ ************************* 
+  1 │ 14            │ ************************************************** 
+  2 │ 17            │ ************************* 
+  3 │ 22            │ ************************* 
+  4 │ 24            │ ************************* 
+  5 │ 28            │ ************************* 
+  6 │ 29            │ ************************* 
+  7 │ 31            │ ************************* 
+  8 │ 37            │ ************************* 
+  9 │ 38            │ ************************* 
+ 10 │ 39            │ ************************* 
+ 11 │ 45            │ ************************* 
+ 12 │ 46            │ *************************************************************************** 
```

Comment:
This is mostly just a nit, but can we use smaller numbers of random numbers so the tables aren't quite so big?  I'm picture someone pulling up the help and not being able to see the top when it prints